composer.json

```diff
@@ -24,6 +24,7 @@
         "embed/embed": "^3.4",
         "erusev/parsedown": "^1.7",
         "fakerphp/faker": "^1.10",
+        "jasny/twig-extensions": "^1.3",
```

Comment:
👍  Love these!

Wonder whether our `localdate` or the `jasny/twig-extensions` one will take precedence in a project?

Reply:
@I-Valchev Good call. I've changed the PR to not add Jasny's DateExtension to prevent the conflicting tag.